### ISSUE TITLE
Multiprocessing leads to disk IO racing w/ model reference 

### ISSUE THREAD

**Issue by tazlin** [Haidra-Org/horde-model-reference#72]

**Comment by db0** [comment]
Isn't this what the multiprocess lock was meant to solve? That's why I had the fasteners.

**Comment by db0** [comment]
Oh this is about the main model reference?

**Comment by tazlin** [comment]
> Oh this is about the main model reference?

Yes; this is the other files pulled from GitHub. It downloads and writes them to disk on start right now, so either I need to skip that or handle it it in some other way.
